cmd/healthbot/counterprobe/counterprobe.go

```diff
@@ -33,7 +33,7 @@ type CounterProbe struct {
 	mLastCounterValue    int64
 	mLastCheck           time.Time
 	mLastSuccessfulCheck time.Time
-	mLatencyHist         syncint64.Histogram
+	mLatencyHist         instrument.Int64Histogram
```

Comment:
You'll this change across a bunch of files. This is the new Histogram type, you'll see that for all other types. For async types you'll see the keyword `Observable` in the middle of the type definition. [See](https://github.com/tablelandnetwork/go-tableland/pull/470/files#diff-7977292631e4d2d29d9b3eb030d818884abd90c3f5766170a7a5af6fc49c28a5R18) how instruments are created.